### PR TITLE
FEP-0 add read me

### PR DESCRIPTION
**Pull request by bareth**

### CODE REVIEW

**Review by utarit — APPROVED** [review]

**Review by rostislav-simonik-plc — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+##  Development
+
+Symlink a package folder during development.
+
+In `eslint-plugin-plentific` repository, run:
+
+```
+yarn link
+```
+Output should look:
```
Comment (fbarbare) [inline README.md:18]:
```suggestion
Output should look like this:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+info You can now run `yarn link "eslint-plugin-plentific"` in the projects where you want to use this package and it will be used instead.
+```
+
+In `dashboard` repository, run:
+
+```
+yarn link eslint-plugin-plentific
+```
+
+Output should look:
```
Comment (fbarbare) [inline README.md:31]:
```suggestion
Output should look like this:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+yarn link eslint-plugin-plentific
+```
+
+Output should look: 
+```
+yarn link vx.x.x
+success Using linked package for "eslint-plugin-plentific".
+```
+
+To reverse this process, simply use `yarn unlink` or `eslint-plugin-plentific` depending on the repository.
```
Comment (fbarbare) [inline README.md:37]:
```suggestion
To reverse this process, simply use `yarn unlink` or `yarn unlink eslint-plugin-plentific` depending on the repository.
```

**Review by fbarbare — APPROVED** [review]
Just a few small changes 😉 

Thank you for this 🙏

File: README.md (hunk 3 above)
Comment (bareth) [reply README.md:37]: eagle eye :D

**Pull request merged**